src/debugger.lisp

```diff
@@ -10,11 +10,38 @@ and printing output to the provided stream. The condition object the debugger
 was entered with and optional command arguments are available for use within
 the command itself."))
 
+(defvar *skip-debugger-prefix* nil
+  "Whether or not to avoid printing the debugger line prefix when
+  executing a command.")
+
+(defmethod run-debugger-command :around (command stream condition &rest arguments)
+  "Prefixes each line of output with semicolons unless told not to."
+  (cond
+    (*skip-debugger-prefix*
+     (call-next-method))
+    (t
+     (let* ((stream (or stream (make-two-way-stream (make-concatenated-stream) (make-broadcast-stream))))
+            (stream-out stream))
```

Comment:
This second variable seems a bit redundant. Why not just `stream`?

Reply:
`pprint-logical-block` would rebind the `stream` variable so I wouldn't be able to use it later in the `make-two-way-stream` call as the input stream; the pretty-printing stream doesn't support input, even if the original stream did, so I have to preserve the original

Reply:
TIL about `pprint-logical-block` rebinding the variable! Thank you.